default/scripting/monster_designs/ShipDesignManifest.focs.txt

```diff
@@ -0,0 +1,36 @@
+ShipDesignManifest
```

Comment:
Just to be sure (I did not notice any cases doing so), I assume the `default` manifest is never modified, only the users(data dir)?